llama_index/vector_stores/mongodb.py

```diff
@@ -203,3 +228,30 @@ def query(self, query: VectorStoreQuery, **kwargs: Any) -> VectorStoreQueryResul
         )
         logger.debug("Result of query: %s", result)
         return result
+
+    def query(self, query: VectorStoreQuery, **kwargs: Any) -> VectorStoreQueryResult:
+        """Query index for top k most similar nodes.
+
+        Args:
+            query: a VectorStoreQuery object
+
+        Returns:
+            A VectorStoreQueryResult containing the results of the query.
+        """
+        if self._use_vectorsearch:
+            try:
+                result = self._query_vectorsearch(query)
+            except OperationFailure as e:
+                # Check for unsupported operation errors
+                if "$vectorSearch" in str(e):
+                    logger.debug(
```

Comment:
Should this be a higher log level?

Reply:
Yes, supposed to be `error`. Fixed, thanks!